northstar/src/api/model.rs

```diff
@@ -10,7 +10,7 @@ use std::{
 };
 
 /// Console configuration
-pub type ConsoleConfiguration = crate::npk::manifest::Console;
+pub type ConsoleConfiguration = crate::npk::manifest::ConsoleQoS;
```

Comment:
Shouldn't `ConsoleConfiguration` include the permissions?

Reply:
yes they do :smile: